packages/verified-fetch/test/cache-control-header.spec.ts

```diff
@@ -70,19 +91,24 @@ describe('cache-control header', () => {
     const oneHourInMs = 1000 * 60 * 60
     const peerId = await createEd25519PeerId()
 
-    // ipns currently only allows customising the lifetime which is also used as the TTL
-    await name.publish(peerId, cid, { lifetime: oneHourInMs })
+    /**
+     * ipns currently only allows customising the lifetime which is also used as the TTL
+     *
+     * lifetime is coming back as 100000 times larger than expected
+     *
+     * @see https://github.com/ipfs/js-ipns/blob/16e0e10682fa9a663e0bb493a44d3e99a5200944/src/index.ts#L200
+     * @see https://github.com/ipfs/js-ipns/pull/308
+     */
+    await name.publish(peerId, cid, { lifetime: oneHourInMs / 100000 })
```

Comment:
It's because we haven't converted IPNS stores it in nanoseconds and we don't convert it back to nanoseconds.
```suggestion
    await name.publish(peerId, cid, { lifetime: oneHourInMs })
```